src/simulation/openables/Implings.ts

```diff
@@ -0,0 +1,329 @@
+		.add('Amulet of power', 3, 1)
+		.add('Ring of forging', 3, 1)
+		.add('Splitbark gauntlets')
+		.add('Mystic boots')
+		.add('Mystic gloves')
+		.add('Rune warhammer')
+		.add('Ring of life', 4)
+		.add('Rune sq shield')
+		.add('Dragon dagger')
+		.add('Nature tiara')
+		.add('Runite bar', 2)
+		.add('Diamond', 4)
+		.add('Pineapple seed')
+		.add('Loop half of key')
+		.add('Tooth half of key')
+		.add('Snapdragon seed')
+		.add('Sinister key')
+		.tertiary(50, 'Clue scroll (hard)')
+});
+
+export const NinjaImpling = new SimpleOpenable({
+	id: 11254,
+	name: 'Ninja impling',
+	aliases: ['ninja impling', 'ninja imp'],
+	table: new LootTable()
+		.add('Snakeskin boots')
+		.add('Splitbark helm')
+		.add('Mystic boots')
+		.add('Rune chainbody')
+		.add('Mystic gloves')
+		.add('Opal machete')
+		.add('Rune claws')
+		.add('Rune scimitar')
+		.add('Dragon dagger(p+)')
+		.add('Rune arrow', 70)
+		.add('Rune dart', 70)
+		.add('Rune knife', 40)
+		.add('Rune thrownaxe', 50)
+		.add('Onyx bolts', 2)
+		.add('Onyx bolt tips', 4)
+		.add('Black dragonhide', 10)
+		.add('Prayer potion(3)', 4)
+		.add('Weapon poison(+)', 4)
+		.add('Dagannoth hide', 3)
+		.tertiary(25, 'Clue scroll (hard)')
+});
+
+export const CrystalImpling = new SimpleOpenable({
+	id: 23768,
+	name: 'Crystal impling',
+	aliases: ['crystal impling', 'crystal imp'],
+	table: new LootTable()
+		.add('Amulet of power', [5, 7])
+		.add('Crystal acorn')
+		.add('Crystal shard', [5, 10])
+		.add('Dragonstone amulet')
+		.add('Dragonstone', 2)
+		.add('Ruby bolt tips', [50, 125])
+		.add('Onyx bolt tips', [6, 10])
+		.add('Rune arrowtips', [150, 250])
+		.add('Rune arrow', [400, 750])
+		.add('Rune javelin heads', [20, 60])
+		.add('Rune dart tip', [25, 75])
+		.add('Rune dart', [50, 100])
+		.add('Dragon dart tip', [10, 15])
+		.add('Dragon dagger', 2)
+		.add('Rune scimitar', [3, 6])
+		.add('Babydragon bones', [75, 125])
+		.add('Ranarr seed', [3, 8])
+		.add('Yew seed')
+		.tertiary(50, 'Clue scroll (elite)')
+		.tertiary(128, 'Elven signet')
+});
+
+export const DragonImpling = new SimpleOpenable({
+	id: 11256,
+	name: 'Dragon impling',
+	aliases: ['dragon impling', 'dragon imp'],
+	table: new LootTable()
+		.add('Dragonstone bolt tips', [10, 30])
+		.add('Dragonstone bolt tips', 36)
+		.add('Mystic robe bottom', 1)
+		.add('Amulet of glory', 3)
+		.add('Dragonstone amulet', 2)
+		.add('Dragon arrow', [100, 250])
+		.add('Dragonstone bolts', [10, 40])
+		.add('Dragon longsword', 1)
+		.add('Dragon dagger(p++)', 3)
+		.add('Dragon dart', [100, 250])
+		.add('Dragonstone', 3)
+		.add('Dragon dart tip', [100, 350])
+		.add('Dragon arrowtips', [100, 350])
+		.add('Dragon javelin heads', [25, 35])
+		.add('Babydragon bones', [100, 300])
+		.add('Dragon bones', [50, 100])
+		.add('Magic seed', 1)
+		.add('Snapdragon seed', 6)
+		.add('Summer pie', 15)
+});
+
+export const LuckyImpling = new SimpleOpenable({
+	id: 19732,
+	name: 'Lucky impling',
+	aliases: ['lucky impling', 'lucky imp'],
```

Comment:
These are also known simply as, `lucky`s if you don't have any other plans for that openable alias